src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala

```diff
@@ -0,0 +1,66 @@
+package io.treeverse.clients
+
+import io.treeverse.committed.Committed.RangeData
+import org.apache.hadoop.fs.Path
+import org.apache.hadoop.io.Text
+import org.apache.hadoop.mapred._
+import org.apache.hadoop.mapreduce.lib.input.FileInputFormat
```

Comment:
I'd rather use (in order of preference):
1. Just org.apache.hadoop.mapreduce, the "new" SDK;
2. Just org.apache.hadoop.mapred, the "old" SDK;
3. Both.
Can we go there?